test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -962,6 +962,16 @@ public void awaitMasterNode(String viaNode, String masterNodeName) {
         safeAwait(listener, TEST_REQUEST_TIMEOUT);
     }
 
+    /**
+     * Waits for all nodes in the cluster to have a consistent view of which node is currently the master.
```

Comment:
A "consistent view" is maybe not the right phrasing as it's not technically consistent (i.e. other cluster states can come in after that moment), but I couldn't come up with a better phrasing. Suggestions are welcome.